src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -618,6 +618,7 @@ public TextWrapping TextWrapping
 				typeof(TextBox),
 				new FrameworkPropertyMetadata(
 					defaultValue: TextWrapping.NoWrap,
+					options: FrameworkPropertyMetadataOptions.AffectsMeasure,
```

Comment:
The code in `OnTextWrapipngChanged` will end up invalidating measure anyways. This probably applies to other properties for TextBox as well.

Not sure if that will end up invalidating measure twice which in turn will cause measuring to happen twice?

Reply:
Afaik it shouldn't, the invalidation is just scheduled, so it should not cause trouble

Reply:
The only effect here is that `InvalidateMeasure` will be called multiple times. In general, we have debouncing conditions where we won't be traversing the tree when asking for invalidation multiple times, so it should not be an issue. That being said, we may want to look for duplicates, particularly if they're done in the property changed handlers because the DP did not have the flag set.